docs/Cloud_integrations/Device_Integration_Center/integrations/graphql.md

```diff
@@ -1,37 +1,42 @@
 # GraphQL
 
-GraphQL is a powerful query language for APIs that enables clients to request only the data they need. 
-*Data Integration Center* allows to send a GraphQL mutation to a server.
+GraphQL is a query language for APIs that enables clients to request only the data they need. 
+**Data Integration Center** allows to send e.g. a mutation to a GraphQL server.
 
 ## Prerequisites
 
+- Basic knowledge of [GraphQL](https://graphql.org/)
 - Working GraphQL server e.g. [Apollo Server](https://www.apollographql.com/docs/apollo-server/v2/getting-started/)
 - GraphQL client e.g. [Sandbox](https://studio.apollographql.com/sandbox/explorer)
 
 ## Create webhook
 
-Adjust body template to send query with mutation. Mutation must be compatible with your schema, so for:
+!!! Info
```

Comment:
Could you add any introduction before the 'info'? It's not the best practice to start a new section with 'info', 'note' 'warning' etc. Some entry describing the process of webhook creation would be required